doc/Type/Cool.pod6

```diff
@@ -1163,12 +1163,18 @@ L<Regex|/type/Regex> as a list of strings.
 
     multi method contains(Cool:D: Str(Cool) $needle, Cool $start? --> Bool:D)
 
+Coerces the first two arguments (in method form, also counting the invocant)
```

Comment:
This should be `Coerces the invocant and $needle`. Not everybody knows by heart what exactly "method form" means.
Is "invocant" the right terminology in Perl6? I would have expected that to be "target", but then that might be the terminology from elsewhere.